nautobot/extras/filters/customfields.py

```diff
@@ -70,7 +70,7 @@ class CustomFieldJSONFilter(CustomFieldFilterMixin, django_filters.Filter):
     """Custom field single value filter for backwards compatibility"""
 
 
-class CustomFieldMultiSelectFilter(CustomFieldFilterMixin, django_filters.Filter):
+class CustomFieldMultiSelectFilter(CustomFieldFilterMixin, django_filters.MultipleChoiceFilter):
     """Custom field single value filter for backwards compatibility"""
```

Comment:
Is the docstring here no longer accurate/relevant? Do we in fact need this to remain a non-multiple-choice filter for some compatibility reason?